librz/asm/p/asm_arm_cs.c

```diff
@@ -16,6 +16,45 @@ typedef struct arm_cs_context_t {
 	int obits;
 } ArmCSContext;
 
+#if CS_NEXT_VERSION < 6
+inline static const char *ARMCondCodeToString(arm_cc cc) {
+	switch (cc) {
+	default:
+		assert(0 && "Unknown condition code");
```

Comment:
add a return to all these asserts. because if it built with `-NDEBUG` then we won't have asserts

Reply:
Of cause, thanks. Funny how switching between Python and C can be real harmful for the C code.
Will fix when the CS PR is merged.